code/create/create-dta.sh

```diff
@@ -0,0 +1,46 @@
+# First argument: Path to install the DTA_study data
+# Second argument: which sourcedata to use
+# Example-call for installing DTA_sourcedata_reduced:
+# >> ./create_BIDS_DTA.sh . /data/BnB_TEMP/Kadelka/DTA_sourcedata_reduced/
+
+# create a dataset
+datalad create -c hirni $1
+cd $1
+
+# installing the necessary code into the dataset
+datalad install -d . -s git@github.com:TobiasKadelka/build_dta.git code/build_dta
+
+# modify the rules for the spec, so the dta-rules will be used
+git config -f .datalad/config --add datalad.hirni.dicom2spec.rules code/build_dta/code/create/dta_dicom2spec_rules.py
+
+# configuring the .datalad/config, so it knows where to find the procedure for renaming the fieldmaps
+# git config -f .datalad/config --add datalad.locations.dataset-procedures code/build_dta/code/procedures
```

Comment:
This is a configuration that belongs to the dataset containing that procedure. You can either make build_dta a datalad dataset and have that config in it (therefore no need for a config call to show up anywhere in the scripts) or you copy the procedures directory into actual DTA dataset (code/procedures for example) and adjust that git-config call accordingly.
If you go for the latter you can call the above mentioned datalad-save afterwards, of course and include both config changes in that one commit.